CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Changed: [#5559](https://github.com/ethereum/aleth/pull/5559) Update peer validation error messages.
 - Fixed: [#5562](https://github.com/ethereum/aleth/pull/5562) Don't send header request messages to peers that haven't sent us Status yet.
 - Changed: [#5568](https://github.com/ethereum/aleth/pull/5568) Improve rlpx handshake log messages and create new rlpx log channel.
+- Added: [#5575](https://github.com/ethereum/aleth/pull/5575) Log active peer count and peer list every 30 seconds.
```

Comment:
> > What do you think about logging peer count to `INFO` verbosity instead?
> > It would help to show in the default logging that the client is doing something even if sync is not progressing. And during sync it's also interesting to see once in a while how many peers we have at the moment.
> > Also in case we have 0 peers `Looking for peers...` could be more user-friendly message than `Active peers: 0`
> 
> I like the idea of logging the peer count with info verbosity, though I'm not sure about the "looking for peers" message. I agree that it's more user friendly but it's not completely accurate since Aleth might not be looking for peers at all. I think I'd prefer to log a 0 peer count and log a "looking for peers"-style message when we're actually looking for peers.

I've added a check in logActivePeers() which logs a "looking for peers" message when there are 0 active peers and discovery is running.